index.html

```diff
@@ -14,27 +14,172 @@
       }
       gtag("js", new Date());
       gtag("config", "G-YJ2WSX508J");
-    </script>
+    </script> -->
 
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta name="description" content="badakhsh portfolio, portfolio" />
 
     <!-- Links -->
     <link href="style.css" rel="stylesheet" />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="project icon"
+      href="./assets/logos/Logo.png"
+      type="image/x-icon"
+    />
+
+    <!-- Script
+    <script src="app.js"></script> -->
```

Comment:
- Kindly remove the unused commented lines of code to make your codebase look cleaner.